app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -19,61 +19,83 @@ inset_text:
+## Teaching qualifications
 
-Fees for full-time courses are usually around £9,250 but you can:
+You can do a postgraduate teacher training course to get:
 
-- get [funding that you do not have to pay back](/funding-and-support/scholarships-and-bursaries) for certain subjects
-- [apply for a student loan](/funding-and-support/tuition-fee-and-maintenance-loans)
+* qualified teacher status (QTS)
+* postgraduate certificate in education (PGCE)
+* qualified teacher status (QTS) with postgraduate certificate in education (PGCE)
 
-Fees are higher for international students.
+You need QTS to teach in most primary, secondary and special schools in England.
 
-You should check the fees for a particular course you are interested in with the course provider.
+You do not need a PGCE to teach, but it can help you:
 
-## Training with a school or groups of schools (fee paying)
-There are different programmes that focus on training you while you teach in a school:
+* get a better understanding of teaching practice and theory
+* earn up to 60 credits towards a master’s - you may want to do a master’s in education, for example, for your professional development
+* teach in other countries
 
-- School-centred initial teacher training (SCITT)
-- School Direct (fee-funded)
+[Find out more about QTS](/what-is-qts).
 
-SCITT programmes are delivered by a network of schools, providing school-based training. You won’t be paid a salary while you are training and will pay fees.
+[Find out more about PGCE](/what-is-a-pgce).
 
-School Direct (fee-funded) programmes are delivered by groups of schools in collaboration with a SCITT or university. The content and structure of a School Direct course is broadly similar to a SCITT course and they don't pay you a salary.
+## Non-salaried teacher training
 
-Some SCITT and School Direct (fee-funded) programmes give you a PGCE or PGDE in addition to QTS. Courses typically take one year but you may be able to study part-time.
+Most teacher training courses are non-salaried. This means that you have to pay course fees and will not earn a salary while you train.
 
-Fees are usually around £9,250 but you can:
+Fees are usually around £9,250, and can be higher for [international students](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
 
-- get [funding that you do not have to pay back](/funding-and-support/scholarships-and-bursaries) for certain subjects
-- [apply for a student loan](/funding-and-support/tuition-fee-and-maintenance-loans)
+If you train to teach certain secondary subjects, you may be eligible for a [scholarship or bursary](/funding-and-support/scholarships-and-bursaries).
 
-You should check the fees for a particular course you are interested in with the course provider.
+No matter what course you choose, you can apply for a student loan.
```

Comment:
Should we say that you can get a student loan for this even if you've already had a student loan previously? (I think that's right - will need checking!). I think this has come up in UR quite a bit - that people don't realise they can get another loan for ITT